### ISSUE TITLE
MacOS 13: MeshAgent full disk access lost when accessing restricted folder

### ISSUE THREAD

**Issue by superbam** [Ylianst/MeshCentral#4823]
MeshCentral 1.1.0, MacOS 13.0.1 on M1 Pro.

After granting full disk access to mesh agent the permissions disappear when trying to use them through Terminal.
![MeshAgent Full Disk Permissions Pre](https://user-images.githubusercontent.com/37909595/204658520-1afa28b7-1a9a-406c-9f6d-1b5bc71fefa4.jpg)
![MeshAgent Full Disk Permissions Attempt](https://user-images.githubusercontent.com/37909595/204658529-e2a6f955-dd4c-45c0-8164-f846163a07c2.jpg)
![MeshAgent Full Disk Permissions Post](https://user-images.githubusercontent.com/37909595/204658540-8bbbad12-6f41-49dc-b985-a93aefe6ce9f.jpg)

**Comment by superbam** [comment]
The universal MacOS agent doesn't seem to have this issue, only the MacOS specific agent.

**Comment by krayon007** [comment]
So you're saying if you use the m1 or x86 specific binary, it has this issue, but not the fat binary?

**Comment by superbam** [comment]
Correct.  Tried it on both Intel and M1 machines.  The Apple OSX Universal agent so far retains full disk access when you try to use it but the macOS Agent (64bit) does not.
